theano/tensor/basic.py

```diff
@@ -1394,11 +1394,14 @@ class Argmax(Op):
     nin = 2  # tensor, axis
     nout = 1
     E_axis = 'invalid axis'
-    __props__ = ()
+    params_type = Generic()
```

Comment:
This interface changes will request some code to allow unpickling old Argmax. The first step is to make a graph with the old node. Then pickle that graph and make a test that unpickle it, compile a Theano function and execute it. Then we need to make that test pass.